tests/phase0/test_sanity.py

```diff
@@ -140,6 +141,32 @@ def test_proposer_slashing(state):
     return state, [block], test_state
 
 
+def test_attester_slashing(state):
+    test_state = deepcopy(state)
+    attester_slashing = get_valid_attester_slashing(state)
+    validator_index = attester_slashing.attestation_1.custody_bit_0_indices[0]
+
+    #
+    # Add to state via block transition
+    #
+    block = build_empty_block_for_next_slot(test_state)
+    block.body.attester_slashings.append(attester_slashing)
+    state_transition(test_state, block)
+
+    assert not state.validator_registry[validator_index].initiated_exit
+    assert not state.validator_registry[validator_index].slashed
+
+    slashed_validator = test_state.validator_registry[validator_index]
+    assert not slashed_validator.initiated_exit
```

Comment:
It seems meaningless check since right now it doesn't trigger `initiate_validator_exit` until we have #850.

Reply:
Just making sure that slashing is following a different path than "initiated_exit"